.config

```diff
@@ -1,3 +1,4 @@
 repo-owner = sanctuary-js
 repo-name = sanctuary-scripts
 contributing-file = .github/CONTRIBUTING.md
+readme-source-files =
```

Comment:
Some projects do not use Transcribe. One can now prevent the readme from being touched by setting `readme-source-files =`. An empty `generate-readme` script is no longer necessary. :)